helm-chart/Chart.yaml

```diff
@@ -59,3 +54,8 @@ dependencies:
     version: 10.24.0
     repository: https://helm.traefik.io/traefik
     condition: traefik.enabled
+
+  - name: keydb
+    version: 0.48.0
+    repository: https://enapter.github.io/charts/
```

Comment:
I see that the [Enapter/charts](https://github.com/Enapter/charts) repository has been archived not too long ago. Perhaps we shouldn't rely on it?

Reply:
Unfortunately there are no alternatives available, this repository is mentioned in the documentation

Reply:
Documentation or not, I don't think we should rely on a know dead project, especially for a key dependency like this. What are we going to do if we need to update the KeyDB version?

The alternative is to add custom templates for the KeyDB resources.

Reply:
How often do we change the version of postgresql or redis dependencies? We can use the repo until we really need to change something. Also, I definitely vote for not including keydb chart in CVAT repository and just forking the repo to cvat organization. Forking and changing a single line in Chart.yaml would take a bit of time and could be done anytime it is really needed. At the moment I don't see any forcing factors to do this.